File: app/src/main/java/igrek/songbook/domain/lyrics/LyricsModel.java
package igrek.songbook.domain.lyrics;

import com.google.common.base.Joiner;

import java.util.ArrayList;
import java.util.List;

public class LyricsModel {
	
	private List<LyricsLine> lines;
	
	public LyricsModel() {
		lines = new ArrayList<>();
	}
	
	public void addLines(List<LyricsLine> lines) {
		this.lines.addAll(lines);
	}
	
	public void addLine(LyricsLine line) {
		lines.add(line);
	}
	
	public List<LyricsLine> getLines() {
		return lines;
	}
	
	@Override
	public String toString() {
		return Joiner.on("\n").join(lines);
	}
}

File: app/src/main/java/igrek/songbook/songpreview/lyrics/LyricsModel.kt
package igrek.songbook.songpreview.lyrics

import com.google.common.base.Joiner

data class LyricsModel(
        val lines: MutableList<LyricsLine> = mutableListOf()
) {

    fun addLines(lines: List<LyricsLine>) {
        this.lines.addAll(lines)
    }

    override fun toString(): String {
        return Joiner.on("\n").join(lines)
    }
}
